Source/Eto.Mac/Eto.XamMac - net45.csproj

```diff
@@ -10,6 +10,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <AssemblyName>Eto.XamMac</AssemblyName>
     <UseMSBuildEngine>True</UseMSBuildEngine>
+    <BaseIntermediateOutputPath>..\..\BuildOutput\obj\Eto.XamMac - net45</BaseIntermediateOutputPath>
```

Comment:
Fixed, tho new line added at the end... some text editros follow the rule of adding a new line at the end while others don't, there is a whole debate on that...